res/sample-mqtt-msg-bus/configuration.toml

```diff
@@ -0,0 +1,158 @@
+    # Also, functions not specified in ExecutionOrder above can still be included which allows for easier
+    # dynamic changes from Consul.
+    [Writable.Pipeline.Functions.FilterByDeviceName]
+      [Writable.Pipeline.Functions.FilterByDeviceName.Parameters]
+        DeviceNames = "Random-Float-Device,Random-Integer-Device"
+    [Writable.Pipeline.Functions.FilterByValueDescriptor]
+      [Writable.Pipeline.Functions.FilterByValueDescriptor.Parameters]
+        ValueDescriptors = "RandomValue_Int8, RandomValue_Int64"
+    [Writable.Pipeline.Functions.TransformToXML]
+    [Writable.Pipeline.Functions.TransformToJSON]
+    [Writable.Pipeline.Functions.CompressWithGZIP]
+    [Writable.Pipeline.Functions.CompressWithZLIB]
+    [Writable.Pipeline.Functions.EncryptWithAES]
+      [Writable.Pipeline.Functions.EncryptWithAES.Parameters]
+        Key = "aquqweoruqwpeoruqwpoeruqwpoierupqoweiurpoqwiuerpqowieurqpowieurpoqiweuroipwqure"
+        InitVector = "123456789012345678901234567890"
+    [Writable.Pipeline.Functions.SetOutputData]
+    [Writable.Pipeline.Functions.MarkAsPushed]
+    [Writable.Pipeline.Functions.PushToCore]
+      [Writable.Pipeline.Functions.PushToCore.Parameters]
+        DeviceName = ""
+        ReadingName = ""
+    [Writable.Pipeline.Functions.HTTPPost]
+      [Writable.Pipeline.Functions.HTTPPost.Parameters]
+        url = "http://"
+        mimeType = "" #OPTIONAL - default application/json
+        persistOnError = "false"
+    [Writable.Pipeline.Functions.HTTPPostJSON]
+      [Writable.Pipeline.Functions.HTTPPostJSON.Parameters]
+        url = "http://"
+        persistOnError = "false"
+    [Writable.Pipeline.Functions.HTTPPostXML]
+      [Writable.Pipeline.Functions.HTTPPostXML.Parameters]
+        url = "http://"
+        persistOnError = "false"
+    [Writable.Pipeline.Functions.MQTTSend]
+      [Writable.Pipeline.Functions.MQTTSend.Parameters]
+        qos="0"
+        key=""
+        autoreconnect="false"
+        retain="false"
+        cert=""
+        persistOnError = "false"
+      [Writable.Pipeline.Functions.MQTTSend.Addressable]
+        Address=   "localhost"
+        Port=      1883
+        Protocol=  "tcp"
+        Publisher= "MyApp"
+        User=      ""
+        Password=  ""
+        Topic=     "sampleTopic"
+
+[Service]
+BootTimeout = "30s"
+ClientMonitor = "15s"
+CheckInterval = "10s"
+Host = "localhost"
+Port = 48095
+Protocol = "http"
+ReadMaxLimit = 100
+StartupMsg = "Sample Configurable Application Service (using MQTT Message Bus) Started"
+Timeout = "5s"
+
+[Registry]
+Host = "localhost"
+Port = 8500
+Type = "consul"
+
+# Database is require when Store and Forward is enabled
+[Database]
+Type = "mongodb"
+Host = "localhost"
+Port = 27017
+Timeout = "30s"
+Username = "appservice"
+Password = "password"
+
+# SecretStore is required when Store and Forward is enabled and running with security
+# so Databse credentails can be pulled from Vault.
```

Comment:
Databse credentails => database credentials

Reply:
You will see the other sample get these 2 comment fixes when your review...